lib/questions.rb

```diff
@@ -187,19 +231,27 @@ def word_count_a_file(file_path)
 # called call_method_from_string('foobar')
 # the method foobar should be invoked
 def call_method_from_string(str_method)
+  send(str_method)
 end
 
 # return true if the date is a uk bank holiday for 2014
 # the list of bank holidays is here:
 # https://www.gov.uk/bank-holidays
 def is_a_2014_bank_holiday?(date)
+  bank_holidays = ['2014, 12, 26', '2014, 12, 25', '2014, 08, 25', '2014, 05, 26', '2014, 05, 05', '2014, 04, 21', '2014, 04, 18', '2014, 01, 01']
+  arr = bank_holidays.map {|i| Date.strptime(i, '%Y, %m, %d')}
+  arr.include?(date)
 end
 
 # given your birthday this year, this method tells you
 # the next year when your birthday will fall on a friday
 # e.g. january 1st, will next be a friday in 2016
 # return the day as a capitalized string like 'Friday'
 def your_birthday_is_on_a_friday_in_the_year(birthday)
+  until birthday.friday?
+    birthday>>(12)
```

Comment:
the `>>` method you want to use is actually a method of `Date` and the `birthday` passed in from the test is a `Time` object 🙄 .

Note also, this would just loop forever as `>>` does not mutate the date, it returns a new one.